miio/integrations/yeelight/specs.yaml

```diff
@@ -80,7 +80,7 @@ yeelink.light.ceiling20:
     supports_color: True
 yeelink.light.ceiling22:
   night_light: True
-  color_temp: [2700, 6500]
+  color_temp: [2600, 6100]
```

Comment:
Are these changes on purpose in this PR?

Reply:
Yes, during I tested on my devices i realized that standard(what I get from yeelight integration) params is wrong. And correct it. If it must be separate PR sorry.

Reply:
It's fine, you could add a short note in the PR description about that though.

Reply:
Name | main.color_temp.min | main.color_temp.max | background.color_temp.min | background.color_temp.max
-- | -- | -- | -- | --
yeelink.light.color3 | 1700 | 6500 | - | -
yeelink.light.color4 | 1700 | 6500 | - | -
yeelink.light.color5 | 1700 | 6500 | - | -
yeelink.light.strip2 | 1700 | 6500 | - | -
yeelink.light.ceiling1 | 2700 | 6500 | - | -
yeelink.light.ceiling20 | 2700 | 6500 | 1700 | 6500
yeelink.light.ceiling4 | 2700 | 6500 | 1700 | 6500
yeelink.light.ceiling22 | 2600 | 6100 | - | -

Result of check.

Reply:
If you happen to know the names, we may want to add them to the yaml file some other point. I'm hoping that we could use the information derived from implementation classes (i.e., an extended version of `_supported_models`) to generate the list of supported devices to the readme/docs.

Reply:
> 

You mean added something like: 
```
yeelink.light.strip2:
          display_name: LED Light Strip 1S
          ...................
```